recipes-core/sysvinit/sysvinit-inittab/start_getty

```diff
@@ -1,28 +1,24 @@
 #!/bin/sh
+# Copyright (c) 2012-2013 National Instruments.
+# All rights reserved.
+# Spawn getty on the enabled, controlling console device. There will be at most
+# one of these. If there aren't any, use /dev/tty0. (This mostly matches
+# sysvinit behavior when $CONSOLE is unset.)
 
-# $1 and $2 are are passed by inittab, its values are given by OE
-CONSOLE="$2"
-SPEED="$1"
+. /etc/natinst/networking/functions.common
+update_all_interface_info
 
```

Comment:
I recall us configuring the CONSOLE in the LVComms images (which is where `start_getty` was used) very intentionally. Can you comment on why the changes we made in this file are no longer necessary, and why `consolegetty` should be preferred in all cases?

Reply:
I wanted to follow what we have in os-common, so I basically replaced start_getty contents with consolegetty. From what I understand, consolegetty does everything start_getty does but also more stuff (printing interface info, replacing CONOLE with DEFAULT_CONSOLE in case it is set to tty0, tty1), so it seemed better to go with consolegetty contents.
Does anyone know why we chose to keep CG, NXG contents different in this case? If we deliberately chose start_getty contents, why didn't we also make the same changes to consolegetty?

Reply:
@rtollert ; since you are probably the only one who understands the black magic we do to keep CONSOLE information sane, can you check out the changes in this PR and comment if anything jumps out at you as obviously wrong?

Reply:
I think @adelcast did all the comms stuff. It was rather deliberately meant to be much closer to upstream IIRC.
The primary value add of consolegetty is that it gives us a way to sub in the serial console if it's enabled. (Updating `/etc/issue` was always a stretch given hotplug.) That never existed for comms (and indeed its value remains a little questionable for PXI).

I don't have a problem with any of these changes.